library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopFederatedIT.java

```diff
@@ -241,11 +241,11 @@ void shouldGetOutgoingEdges() {
         // Then
         assertThat(result)
                 .extracting(item -> item.id().toString())
-                .containsExactly("[p1, p2]", "[p1, s1]");
+                .containsExactlyInAnyOrder("[p1, p2]", "[p1, s1]");
```

Comment:
Why is this change required? Does this mean that it was in a defined order before but now no longer is?

Reply:
@GCHQDeveloper314 changing the filtering appears to have just swapped the order of the returned edges. If the order doesn't matter then tempted to leave as is

Reply:
Ok, that's probably fine then.